### ISSUE TITLE
Make available via maven repository

### ISSUE THREAD

**Issue by gigaSproule** [nguillaumin/jflac#7]
Is it possible to make this available via a maven repository? e.g. bintray.com?

**Comment by nguillaumin** [comment]
It's certainly possible, I've published artifacts on Maven Central (where my preference would be) via [Sonatype OSS](http://central.sonatype.org/pages/ossrh-guide.html), but there are a few difficulties:

* It's just a fork of the [original project](http://jflac.sourceforge.net/), so they should be the one publishing it
* Not sure what coordinates to publish that under:
 * `jflac:jflac` would be consistent with the POM but I suspect it would not be accepted by Sonatype / Central as it needs to be a reverse package name or domain name.
 * `org.kc7bf.jflac:jflac`would be consistent with the package name, but a bit cryptic and I don't really own this package, it's from the original authors
 * `net.sourceforge.jflac:jflac`would be consistent with the original project, but at the same time the package would not be build from their sources but mine
 * Something like `com.github.nguillaumin:jflac` would be consistent with this project location, but not very discoverable, and may confuse people if the original authors end up publishing it as well
* Possibly rename the package `org.jflac` then publish it under this name, but that would feel like stealing it from the original authors so I'd need their approval.

Happy to hear suggestions?

So to sum up, I'd prefer that the version that is published is an "official" one, either published by the jFLAC project, or if they're happy to make this project the official one.

I'll try to get in touch with them to ask them opinion.

**Comment by nguillaumin** [comment]
Posted on their ML: https://sourceforge.net/p/jflac/mailman/message/35300318/

**Comment by gigaSproule** [comment]
I didn't realise this was a fork of the original project. But I totally agree that the original guys should have a say in this first. If they're not willing to take this forward, then I would say go with `org.jflac` as that seems to be the cleanest.

If they aren't willing to take this further, then I can't see what more we can do than to make them aware.

**Comment by nguillaumin** [comment]
Agreed. I think there's little chance they'll respond to my message, since the ML have been dead since 2012 or something, but we'll see.

**Comment by nguillaumin** [comment]
There you go: http://search.maven.org/#search%7Cga%7C1%7Cjflac
